paneldb/server/templates/baitsets.html

```diff
@@ -19,9 +19,32 @@ <h4>Upload a new bait set</h4>
       <textarea class="form-control" id="exampleFormControlTextarea1" rows="2" ></textarea>
     </div>
 
+
+    <!-- Genomic build dropdown -->
+    <div class="dropdown">
```

Comment:
This shouldn't be a dropdown object, but a select (look here: https://getbootstrap.com/docs/4.1/components/input-group/#custom-select)

I'd write the form code like this:
```
<form action="{{ url_for('baitsets') }}" method="POST" id="submit_baitset" name="submit_baitset" enctype=multipart/form-data>
    <div class="form-group">
      <br>
      <h4>Upload a new bait set</h4>
      <div class="form-check form-check-inline col-8">
        <div class="input-group-prepend">
          <span class="input-group-text" id="inputGroup-sizing-default">Bait set name</span>
        </div>
        <input type="text" class="form-control" aria-label="baitset_name" aria-describedby="inputGroup-sizing-default" required>
      </div>
      <div class="form-check form-check-inline">
        <span class="input-group-text" id="inputGroup-sizing-default">Chromosome build</span>
          <select class="form-control" id="chr_build">
            <option>GRCh37</option>
            <option>GRCh38</option>
          </select>
      </div>
    </div>

    <div class="form-group">
      <label for="exampleFormControlTextarea1">Description</label>
      <textarea class="form-control" id="exampleFormControlTextarea1" rows="2" ></textarea>
    </div>

    <div class="input-group mb-3">
      <div class="custom-file">
        <input type="file" accept=".bed, .txt, .csv" class="custom-file-input" name="inputFile" id="inputfile" onchange="setfilename(this.value);" required>
        <label class="custom-file-label" id="baiset_file_label" for="inputFile">Choose file</label>
      </div>
    </div>
      <button type="submit" class="btn btn-primary">Submit</button>
  </div>
</form>
```

Read about radio buttons here: https://getbootstrap.com/docs/4.0/components/forms/
So you can introduce the option "Create new baitset" and "New baitset version" as we discussed yesterday